CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.12.1 / 2022-01-29
+
+* [BUGFIX] Make the Go 1.17 collector thread-safe #969
```

Comment:
Nit: I would call it "concurrency-safe". "Thread" is a rather specific term, and the Go runtime, while utilizing threads, pretty much decouples Goroutines from threads. (AFAIK, even on a single-CPU single-threaded machine you could still run into data races.)